app/uk/gov/hmrc/tariffclassificationfrontend/controllers/RulingController.scala

```diff
@@ -44,12 +45,17 @@ class RulingController @Inject()(authenticatedAction: AuthenticatedAction,
 
   private lazy val menuTitle = CaseDetailPage.RULING
 
+  private def editRuling(f: Form[DecisionFormData], c: Case)
```

Comment:
This code was repeated twice. 
I think now the code here is more readable and maintainable.